code/game/gamemodes/cult/ritual.dm

```diff
@@ -0,0 +1,594 @@
+//This file was auto-corrected by findeclaration.exe on 25.5.2012 20:42:32
```

Comment:
Where did that come from?

Reply:
Not sure how that reappeared with the merge. Removed it again.